build.sbt

```diff
@@ -26,20 +31,6 @@ lazy val guiSettings: Seq[sbt.Def.Setting[_]] = Seq(
   routesGenerator := InjectedRoutesGenerator
 )
 
-// TODO: why do we need to define this explicitly
-// this is the root project, aggregating all sub projects
-lazy val root = Project(
-    id   = "root",
-    base = file("."),
-    // configure your native packaging settings here
-    settings = Seq(
-        maintainer           := "Adriaan Moors <adriaan@lightbend.com>",
-        packageDescription   := "Scala Bot",
-        packageSummary       := "Automates stuff on Github"),
-    // always run all commands on each sub project
-    aggregate = Seq(core, amazon, github, cli, jenkins, server)
```

Comment:
the omission of `lightbend` here was just an oversight (mine), so it comes down to the status of `gui`